lib/peep.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+
```

Comment:
overall the code and test quality looks great, really easy to read and understand exactly what is going on, I can follow the code and work out in my head what a user would interact with and how the info is stored and recalled in the database, really good!